### DISCUSSION TITLE
Custom links in new window

### DISCUSSION THREAD

**Discussion by mbruner** [argoproj/argo-workflows#7977] [General]
Hello community!

We use custom links and really like this feature. Simple usability question: how can we make these links opened in a new window/tab?

**Comment by RoryDoherty** [comment]
Not sure if you got the answer to this or not but if you ctrl + click on the link it will open in a new tab instead of in the current window

PR opened for this in 2020 https://github.com/argoproj/argo-workflows/pull/2935
